spec/section_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'helper'
+require_relative '../lib/osu-cc-scraper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.